grails-app/controllers/au/org/ala/biocollect/merit/ProxyController.groovy

```diff
@@ -35,11 +35,11 @@ class ProxyController {
     }
 
     def features(){
-        render webService.get("${grailsApplication.config.spatial.layersUrl}/objects/${params.layerId}", false)
+        render webService.get("${grailsApplication.config.spatial.layersUrl}/objects/${params.layerId}")
     }
 
     def feature(){
-        render webService.get("${grailsApplication.config.spatial.layersUrl}/object/${params.featureId}", false)
+        render webService.get("${grailsApplication.config.spatial.layersUrl}/object/${params.featureId}")
```

Comment:
same comment as above

Reply:
Reverted in 14fc0d1db14c53cf1143e63d87ffe42f8fd13313